decode/src/lib.rs

```diff
@@ -968,7 +961,7 @@ mod tests {
             },
         };
         let expected = "{\"identifier\":\"qsfp_plus_sff8636\",\"vendor\":\
-            {\"name\":\"foo\",\"oui\":\"a8-40-25\",\"part\":\"bar\",\
```

Comment:
Are changes in this file intentionally part of the PR?

Reply:
Yeah, they are. Kind of piggy-backing, but it is intentional.